### DISCUSSION TITLE
Monetization Options

### DISCUSSION THREAD

**Discussion by EthanLipnik** [EthanLipnik/OpenSesame#5] [Q&A]
Hey! I'm looking through my options on monetizing this app to keep maintaining and adding to it especially as it starts to support more platforms. I'd love to hear you guys' ideas on the best way you would prefer it.

I'm considering either a GitHub sponsor button and/or a tip jar in the app.

Another option was a "pro" option with 10 free accounts and anymore would require that, but I don't really want to lock an open source project's App Store release behind a paywall.

What do you guys think?

**Comment by JohJakob** [comment]
I would add both sponsoring on GitHub and a tip jar inside the app. GitHub sponsoring would be a great option to get (monthly) financial support from interested developers, whereas a tip jar would focus more on the users and one-time payments.

**Reply by EthanLipnik** [reply]
Probably going to be the best solution.

**Comment by EthanLipnik** [comment]
Update: Added a Ko-fi. Will add native GitHub donations soon
